gcs/CONFIGURATION.md

```diff
@@ -447,10 +445,10 @@ default service account impersonation.
     modifications made outside this connector instance may not be immediately
     reflected.
 
-*   `fs.gs.performance.cache.max.entry.age.ms` (default: `5000`)
```

Comment:
if someone do not provide the unit, it will be taken as ms? i.e. Will customer need to change their configs once they migration to 3.0?

Reply:
Yes, it will be treated as `ms` - there are unit test for this too.